corehq/apps/hqwebapp/templatetags/hq_shared_tags.py

```diff
@@ -411,3 +411,16 @@ def prelogin_url(context, urlname):
         return reverse(urlname, args=[context['LANGUAGE_CODE']])
     else:
         return reverse(urlname)
+
+
+@register.simple_tag(takes_context=True)
+def url_replace(context, field, value):
```

Comment:
I know I'm late to the game, but I would have called this something like `current_url_params_with_one_param_replaced`. To me short names mean "this is used all the time" and long names mean "this does exactly the thing it says, and is probably just used in one or two places to achieve that specific result", and this feels like the latter.